### DISCUSSION TITLE
Ddclient integration

### DISCUSSION THREAD

**Discussion by linux-universe** [cockpit-project/cockpit#20055] [Ideas]
it would be nice if you could graphically configure a dyndns update in cockpit via ddclient.

**Comment by martinpitt** [comment]
This isn't something that the cockpit team will do themselves, but of course anyone is welcome to create such a page. Please see https://cockpit-project.org/blog/creating-plugins-for-the-cockpit-user-interface.html and https://cockpit-project.org/blog/cockpit-starter-kit.html for the initial steps.
